test/functional/feature_filelock.py

```diff
@@ -15,6 +15,10 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 2
 
+    # TODO: remove it after 20267 done
```

Comment:
nit: Consider squashing all bitcoin#20267-related TODOs into one commit, should make it easier to revert (same for TODOs hinging on other backports)

Reply:
grouped 4 commits for 20267